apache/superset
# feat(homepage): add more cards and new layout

### SUMMARY
This pr allows more cards to be shown on the welcome page and by adding responsive layout for the cards. Now user can see a total of 4 cards for each row of user data.

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF

after

https://user-images.githubusercontent.com/17326228/127382738-c9d226f5-e5ce-4974-8c1c-f8828da7993d.mov


resizing 

https://user-images.githubusercontent.com/17326228/127383290-ae0d5ff2-a892-441d-abc3-c37097830d89.mov

empty state
<img width="1318" alt="Screen Shot 2021-07-26 at 7 58 28 AM" src="https://user-images.githubusercontent.com/17326228/127036426-0e49aae8-6664-4dd7-b10e-10524142adda.png">


### TESTING INSTRUCTIONS
Go to Welcome page and test out the layout of the cards.

For first time user experience:
1. Create new ephemeral env or wipe out you local database  and spin up the a new superset instance and view the new layout.
2. The layout should show charts and dashboard table open set on examples.

For existing user:
Open workspace or existing env and the examples tabs should exist in the dashboard and table but not necessarily set on them if user has localstorage set on already clicked tabs. 

### ADDITIONAL INFORMATION
